sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/externalrules/CxxExternalRulesSensor.java

```diff
@@ -32,22 +38,29 @@
 import org.sonar.plugins.cxx.CxxLanguage;
 import org.sonar.plugins.cxx.utils.CxxMetrics;
 import org.sonar.plugins.cxx.utils.CxxReportSensor;
+import org.sonar.plugins.cxx.utils.CxxUtils;
 import org.sonar.plugins.cxx.utils.StaxParser;
 
 /**
  * Custom Rule Import, all static analysis are supported.
  *
- * @author jorge costa
+ * @author jorge costa, stefan weiser
  */
 public class CxxExternalRulesSensor extends CxxReportSensor {
   public static final Logger LOG = Loggers.get(CxxExternalRulesSensor.class);
   public static final String REPORT_PATH_KEY = "sonar.cxx.other.reportPath";
+  public static final String SONAR_CXX_XSLT_KEY = "sonar.cxx.xslt.";
```

Comment:
In case we keep it for 'other' name must be  sonar.cxx.other.xslt.xxx. 

Think  it's not hard to change later one. Let's start as part of 'other'. There is also no problem with the sequence then.